docs/modules/optim.rst

```diff
@@ -26,9 +29,9 @@
 
 .. raw:: html
 
-  CollaborativeOptimizer is a legacy version of hivemind.Optimizer. **For new projects, please use hivemind.Optimizer.**
+  CollaborativeOptimizer is a legacy version of hivemind.Optimizer. <b>For new projects please use hivemind.Optimizer</b>.
   Currently, hivemind.Optimizer supports all the features of CollaborativeOptimizer and then some.
```

Comment:
> supports all the features of CollaborativeOptimizer and then some.

The sentence seems to be not finished.

Reply:
I respectfully disagree [1]
[1] - https://www.urbandictionary.com/define.php?term=and%20then%20some

Reply:
I think this phrase is not widely-known enough to be used in the docs. Most readers just won't understand it :(